src/WebJobs.Script.WebHost/host.json

```diff
@@ -0,0 +1,7 @@
+{
+  "version": "2.0",
+  "extensionBundle": {
```

Comment:
You don't mean to add this file right? Remove.

Reply:
Removed.